src/NumericalAlgorithms/Interpolation/IrregularInterpolant.hpp

```diff
@@ -0,0 +1,104 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Declares functions for interpolating variables to a set of points.
```

Comment:
This comment is no longer correct (not a function anymore), and I think given the file name and class name could just be removed